raijin_scripts/deploy/build_environment_module.py

```diff
@@ -106,6 +102,13 @@ def date(date_format="%Y%m%d") -> str:
     return datetime.datetime.now().strftime(date_format)
 
 
+def _log_output(line):
+    try:
+        LOG.info(line.encode('ascii').decode('utf-8'))
```

Comment:
This line, and this whole function, looks suspiciously unnecessary. At the very least it needs comments explaining what it is achieving by attempting to encode to ascii and decode to utf-8.
The except case is also extremely weird.
It isn't important enough to block, but we need to avoid this sort of thing or the code very quickly becomes unmaintainable...